adapters/oidc/undertow/src/main/java/org/keycloak/adapters/undertow/KeycloakServletExtension.java

```diff
@@ -174,7 +174,10 @@ public Account verify(Credential credential) {
         });
 
         log.debug("Setting jsession cookie path to: " + deploymentInfo.getContextPath());
-        ServletSessionConfig cookieConfig = new ServletSessionConfig();
+        ServletSessionConfig cookieConfig = deploymentInfo.getServletSessionConfig();
+        if (cookieConfig == null) {
+            cookieConfig = new ServletSessionConfig();
+        }
         cookieConfig.setPath(deploymentInfo.getContextPath());
```

Comment:
Should we set the path for the session config if it's already created? Doesn't seem correct